src/client/containers/ProfilePage/MentorProfile/MentorProfile.component.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import { Avatar } from '../../../components/Avatar/Avatar';
+import mentor from '../../../assets/images/mentor.png';
+import Vector from '../../../components/Appheader/Vector.svg';
+import Label from '../../../components/Label/Tags/Label';
+import DropDown from '../../../components/Dropdown/DropDown';
+import Footer from '../../../components/footer/footer';
+import { Button } from '../../../components/Button/Button';
+import { TimeZoneDropDown } from '../../../components/TimeZone/TimeZone.component';
+import './MentorProfile.style.css';
+
+const skills = [
```

Comment:
Skills should be obtained from API. Not sure what you intended here with storybook for a container. Will you create another file for the container which also fetches data from API?